src/components/NotFoundPage.css

```diff
@@ -0,0 +1,31 @@
+.not-found-container {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	height: 100vh;
+	width: 100%;
+}
+
+.not-found-heading {
+	font-size: 4.5em;
+	margin-bottom: 0;
+}
+
+.Nav-link {
+	--color-text: var(--color-cobalt-blue);
```

Comment:
The blue is beautiful, but not accessible in light or dark mode. I know you are all still making decisions regarding color schemes, especially for dark mode, but I wanted to point that out just in case you were envisioning this as the final color. 
`light mode`
<img width="641" alt="Screenshot 2024-04-03 at 4 32 51 PM" src="https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/97368143/8229d7d3-cd14-430e-b876-40d38b376808">
`dark mode`
<img width="369" alt="Screenshot 2024-04-03 at 4 34 58 PM" src="https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/97368143/03e5c7a4-e0a2-4128-be5c-015cd30182f5">

Reply:
Hi @polly89 ! Thank for addressing this issue! The color of the text will be white for a dark mode, and black for the white light mode. @jaguilar89 is taking care of this in his PR related to toggle dark and white mode.
<img width="1268" alt="Screenshot 2024-04-03 at 2 51 27 PM" src="https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/88466266/d6772b80-43af-4694-907d-cfcd58fc8a53">

<img width="1240" alt="Screenshot 2024-04-03 at 2 51 39 PM" src="https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/88466266/cc647100-b03d-4ecd-a913-3240d0b794da">

Reply:
That's perfect... I figured that would be the case once I saw Jose's issue. Sorry for the confusion.